### ISSUE TITLE
Invalid `textAlign` type

### ISSUE THREAD

**Issue by nezort11** [Faire/mjml-react#82]
```html
<MjmlSection textAlign="start">
   ...
</MjmlSection>
```

With `validationLevel: "strict"` gives:

```
Error: ValidationError: 
 ... (mj-section) — Attribute text-align has invalid value: start for type Enum, only accepts left, center, right
```

*Probably `start`/`end` throw ValidationError for most client compatibility: https://caniuse.com/mdn-css_properties_text-align_flow_relative_values_start_and_end

**Comment by IanEdington** [comment]
Thanks for reporting and putting in a fix @nezort11!!

This seems like it should have been picked up by the component import from `mjml`. It will have to changed in the generate-mjml script. There's probably some discovery we'll have to do to figure out why this isn't working as expected. We'll try to prioritize this ticket

**Comment by IanEdington** [comment]
fixed by #85
